pkg/datapath/loader/netlink.go

```diff
@@ -316,17 +316,22 @@ func setupVxlanDevice(sysctl sysctl.Sysctl, port, srcPortLow, srcPortHigh uint16
 // taken control of the encapsulation stack on the node, as it currently doesn't
 // explicitly support sharing it with other tools/CNIs. Fallback devices are left
 // unused for production traffic. Only devices that were explicitly created are used.
-func setupIPIPDevices(sysctl sysctl.Sysctl, ipv4, ipv6 bool) error {
+func setupIPIPDevices(sysctl sysctl.Sysctl, ipv4, ipv6 bool, mtu int) error {
 	// FlowBased sets IFLA_IPTUN_COLLECT_METADATA, the equivalent of 'ip link add
 	// ... type ipip/ip6tnl external'. This is needed so bpf programs can use
 	// bpf_skb_[gs]et_tunnel_key() on packets flowing through tunnels.
-
 	if ipv4 {
-		// Set up IPv4 tunnel device if requested.
-		if _, err := ensureDevice(sysctl, &netlink.Iptun{
-			LinkAttrs: netlink.LinkAttrs{Name: defaults.IPIPv4Device},
+		// sizeof(struct iphdr) for the IPIP encap.
+		ipipOverhead := 20
```

Comment:
We should also reflect these numbers in https://github.com/cilium/cilium/blob/main/pkg/mtu/mtu.go#L140-L148 so that anyone relying on `RouteMTU` to make packets that should fit inside tunnels is also aware.

Reply:
I'll follow-up with a comment. Right now IPIP != vxlan/geneve and we only use the former to terminate external traffic, but Pods don't send traffic via IPIP like in case of vxlan/geneve. So code-wise it's fine as-is, but I'll update the comments in a follow-up to note that down there.